frontend/src/views/web/login.html

```diff
@@ -15,15 +15,11 @@
         <label for="password" >Password (min 8 characters) *</label>
         <div class="wrn-msg text-highlight" ng-show="auth.isValid.password">{{auth.wrnMsg.password}}</div>
     </div>
-    <div class="row">
-    	<div class="col s6 align-left rm-gut">
-	    	<input type="checkbox" class="filled-in" ng-model="auth.isRem" id="remeb">
-			<label for="remeb">Remember Me</label>
-		</div>
+
 		<div class="col s6 align-right right rm-gut">
 	    	<a ui-sref="auth.reset" class="light-link fg-pass">Forgotten Account?</a>
-		</div>	
-    </div>
+		</div>
+
     <div class="align-left">
     	<button class="waves-effect waves-dark btn ev-btn-dark w-300" type="submit" value="Submit">LogIn</button>
         <span ng-hide="auth.isConfirm" class="fade"> &nbsp;{{auth.wrnMsg.cred}}</span>
```

Comment:
@anweshknayak : please make "auth.wrnMsg.cred" a div and place it outside of Login container. I hope you understood, if not then let me know please. Thanks.